forest/cli/src/cli/snapshot_cmd.rs

```diff
@@ -0,0 +1,217 @@
+                ]);
+                let output_path = match strfmt(&output_path.display().to_string(), &vars) {
+                    Ok(path) => path.into(),
+                    Err(e) => {
+                        cli_error_and_die(format!("Unparsable string error: {}", e), 1);
+                    }
+                };
+
+                let params = (
+                    epoch,
+                    *recent_stateroots,
+                    *include_old_messages,
+                    output_path,
+                    TipsetKeysJson(chain_head.key().clone()),
+                    *skip_checksum,
+                );
+
+                // infallible unwrap
+                let out = chain_export(params).await.map_err(handle_rpc_err).unwrap();
+
+                println!("Export completed. Snapshot located at {}", out.display());
+            }
+            Self::Fetch { snapshot_dir } => {
+                let snapshot_dir = snapshot_dir
+                    .clone()
+                    .unwrap_or_else(|| default_snapshot_dir(&config));
+                match snapshot_fetch(&snapshot_dir, config).await {
+                    Ok(out) => println!("Snapshot successfully downloaded at {}", out.display()),
+                    Err(e) => cli_error_and_die(format!("Failed fetching the snapshot: {e}"), 1),
+                }
+            }
+            Self::Dir => {
+                let dir = default_snapshot_dir(&config);
+                println!("Default snapshot dir: {:?}", dir);
+            }
+            Self::List { snapshot_dir } => {
+                let snapshot_dir = snapshot_dir
+                    .clone()
+                    .unwrap_or_else(|| default_snapshot_dir(&config));
+                println!("Snapshot dir: {:?}", snapshot_dir);
+                if let Ok(dir) = fs::read_dir(snapshot_dir) {
+                    println!("\nLocal snapshots:");
+                    for entry in dir.flatten() {
+                        let path = entry.path();
+                        if path.is_file() {
+                            if let Some(filename) = entry.file_name().to_str() {
+                                if filename.ends_with(".car") {
+                                    println!("{filename}");
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+            Self::Remove {
+                filename,
+                snapshot_dir,
+                yes,
+            } => {
+                let yes = yes.unwrap_or_default();
+                let snapshot_dir = snapshot_dir
+                    .clone()
+                    .unwrap_or_else(|| default_snapshot_dir(&config));
+                let mut snapshot_path = snapshot_dir;
+                snapshot_path.push(filename);
+                if snapshot_path.exists()
+                    && snapshot_path.is_file()
+                    && snapshot_path.extension() == Some(OsStr::new("car"))
```

Comment:
Don't we need to remove the associated `.sha256sum` if there is one?

Reply:
Good catch, will handle that

Reply:
fixed